src/main/java/AddList.java

```diff
@@ -0,0 +1,56 @@
+public class AddList {
+
+    private final String[] entries = new String[100]; 
+    private final boolean[] done = new boolean[100];   
+    private static final String SEPARATOR = "\t ___________________________"; 
+    private int index = 0; 
+
+    public void addEntry(String entry) {
+        if (index < entries.length) {
+            entries[index] = entry; 
+            index++; 
+            System.out.println(SEPARATOR); 
+            System.out.println("\t added: " + entry); 
+            System.out.println(SEPARATOR); 
+        } else {
+            System.out.println("\t Cannot add entry"); 
+        }
+    }
+
+    public void displayEntries() {
+        System.out.println(SEPARATOR); 
+        if (index == 0) {
```

Comment:
Perhaps you use a guard clause here instead. Referring to the "make the happy path prominent" code quality, u could use if (index != 0) as a guard clause and move your current else block into this if block. That might make your code neater and easier to read.